src/rtc_monitor.ts

```diff
@@ -117,6 +123,10 @@ export class RTCMonitor extends EventEmitter {
                 continue;
             }
 
+            if (localOut[ssrc].packetsSent === this.stats.lastLocalOut[ssrc]?.packetsSent) {
+                continue;
+            }
```

Comment:
This is the actual fix. Everything else is needed boilerplate to get the info needed to assess this.